chef/cookbooks/provisioner/templates/default/autoyast-upgrade.xml.erb

```diff
@@ -64,25 +64,18 @@
           key_re='crowbar\.install\.key=([^ ]+)'
           if [[ $(cat /proc/cmdline) =~ $key_re ]]; then
               export CROWBAR_KEY="${BASH_REMATCH[1]}"
-              echo "$CROWBAR_KEY" >/mnt/etc/crowbar.install.key
-          elif [[ -f /mnt/etc/crowbar.install.key ]]; then
-              export CROWBAR_KEY="$(cat /mnt/etc/crowbar.install.key)"
+              echo "$CROWBAR_KEY" >/etc/crowbar.install.key
+          elif [[ -f /etc/crowbar.install.key ]]; then
+              export CROWBAR_KEY="$(cat /etc/crowbar.install.key)"
           fi
-          post_state() {
-              mkdir -p /var/log/crowbar
-              local curlargs=(-o /dev/null --connect-timeout 60 -s \
-                -L -X POST --data-binary "{ \"name\": \"$1\", \"state\": \"$2\" }" \
-                -H "Accept: application/json" -H "Content-Type: application/json" \
-                --max-time 240)
-              [[ $CROWBAR_KEY ]] && curlargs+=(-u "$CROWBAR_KEY" --digest --anyauth)
-              curl "${curlargs[@]}" "http://$IP/crowbar/crowbar/1.0/transition/default"
-          }
 
-          curl -s -o /mnt/usr/sbin/crowbar_join <%= @crowbar_join %>
-          chmod +x /mnt/usr/sbin/crowbar_join
+          export PASS="$(sed -e 's/^machine-install://' <<< $CROWBAR_KEY)"
```

Comment:
Do we need this line?

Reply:
Oh, I forgot the block with writing `/etc/crowbarrc`. Or could we already assume it is there when `autoyast-upgrade.xml.erb` is used?

Reply:
So I was wondering about the upgrade case. Not sure how it's handled. Maybe this is the right place?

Reply:
Yeah, so I've added `/etc/crowbarrc` here as well, just to be sure it is present.